Documentation/Maintenance/Release.md

```diff
@@ -853,6 +867,9 @@ For the final release, the release notes produced should be used to
   * Add a release note doc in [ITK/Documentation/ReleaseNotes](https://github.com/InsightSoftwareConsortium/ITK/tree/master/Documentation/ReleaseNotes)
   * Create a post in ITK project on [ResearchGate]
   * Update [ITK's Wikipedia page](https://en.wikipedia.org/wiki/Insight_Segmentation_and_Registration_Toolkit).
+  * Send out a summary to Arliss at NumFOCUS.
```

Comment:
Does Arliss have a generic address or job title so that if she leaves we at least have the email address or job title (we may not want to make here the email address public)? Put a link to NumFOCUS or their contact people website?

Reply:
We can use `help@numfocus.org`